manager/assets/modext/core/modx.js

```diff
@@ -95,6 +96,14 @@ Ext.extend(MODx,Ext.Component,{
         });
     }
 
+    ,initAutocompletePagetitle: function() {
+        Ext.ComponentMgr.onAvailable("modx-resource-pagetitle", function(field) {
+            var setting_pagetitle_autocomplete = +MODx.config.resource_pagetitle_autocomplete;
+            var autocomplete = setting_pagetitle_autocomplete ? "on" : "off";
+            field.defaultAutoCreate = {tag: "input", type: "text", size: "20", autocomplete: autocomplete, msgTarget: 'under' }
```

Comment:
Why the size is 20?

Reply:
I copied from here (default code of modx.js)
![image](https://user-images.githubusercontent.com/8276064/55462974-b4834000-5600-11e9-9a19-bc8accda96e6.png)

and change only logic for autocomplete

Reply:
i think (if we can write more than 20 characters in pagetitle field - 'size' is only default, but overrides by ExtJS for everyfied with strict 'size' option)